pkg/network/usm/sharedlibraries/watcher_test.go

```diff
@@ -37,13 +38,32 @@ import (
 	"github.com/DataDog/datadog-agent/pkg/util/log"
 )
 
+func waitForProcessMonitor(t *testing.T, pm *monitor.ProcessMonitor) {
```

Comment:
why not exporting the waitForProcessMonitor in the process_monitor test and using that? as it the same function?
Maybe moving it to utils package and using it for both